pyaedt/edb_core/padstack.py

```diff
@@ -470,6 +470,51 @@ def get_pad_parameters(self, pin, layername, pad_type=0):
         rot = padparams[5].ToDouble()
         return geom_type, parameters, offset_x, offset_y, rot
 
+    @pyaedt_function_handler
+    def set_all_antipad_value(self, value):
+        """Set all anti-pads from all pads-tack definition to the given value.
+
+        Parameters
+        ----------
+        value float, str
+            anti-pad value
+
+        Returns
+        --------
+        bool
+            ``True`` when all succeed ``False`` if one of anti-pads value failed to be assigned.
+        """
+        if self.padstacks:
+            for padstack in list(self.padstacks.values()):
+                cloned_padstack_data = self._edb.Definition.PadstackDefData(padstack.edb_padstack.GetData())
+                layers_name = cloned_padstack_data.GetLayerNames()
+                all_succeed = True
+                for lay in layers_name:
+                    geom_type, parameters, offset_x, offset_y, rot = self.get_pad_parameters(
+                        padstack.edb_padstack, lay, 1
+                    )
+                    if geom_type == 1:
+                        params = convert_py_list_to_net_list([self._pedb.edb_value(value)] * len(parameters))
+                        geom = self._edb.Definition.PadGeometryType.Circle
+                        offset_x = self._pedb.edb_value(offset_x)
+                        offset_y = self._pedb.edb_value(offset_y)
+                        rot = self._pedb.edb_value(rot)
+                        antipad = self._edb.Definition.PadType.AntiPad
+                        if cloned_padstack_data.SetPadParameters(lay, antipad, geom, params, offset_x, offset_y, rot):
+                            self._logger.info(
+                                "Pad-stack definition {}, anti-pad on layer {}, has been set to {}".format(
+                                    padstack.edb_padstack.GetName(), lay, str(value)
+                                )
+                            )
+                        else:
+                            self._logger.error(
+                                "Failed to reassign anti-pad value {} on Pads-stack definition {},"
+                                " layer{}".format(str(value), padstack.edb_padstack.GetName(), lay)
+                            )
+                            all_succeed = False
```

Comment:
Here, we could return `False`.
```suggestion
                            return False
```

Reply:
if we return False it will stop the loop although I wanted to keep running the loop. Siince the commandis called set_all_antipad I wanted o notice the user something wrong happened  but not necessarily stop the loop over all padstack definition. So I will keep like that for now